README.md

```diff
@@ -53,17 +53,35 @@ To get a local copy up and running follow these steps.
    php bin/console hautelook:fixtures:load
    ```
 
-6. ????
-
-7. Profit
-
 You should now be able to browse to the application
 
 ```sh
 symfony server:start -d
 ```
 
 ## Admin user
-A user with admin permissions gets created when fixtures are run. It's credentials are:  
-Username: admin@admin.com  
+
+A user with admin permissions gets created when fixtures are run.
+It's credentials are:\
+Username: admin@admin.com\
 Password: 123123
+
+## Coding standards
+
+Apply coding standards to php-files:
+
+```sh
+docker-compose exec phpfpm composer apply-coding-standards
+```
+
+Check markdown docs:
+
+```sh
+yarn markdownlint README.md docs
```

Comment:
Use a command alias in `package.json` for this (cf. `coding-standards-apply/markdownlint`).